src/main/resources/portal.properties.EXAMPLE

```diff
@@ -334,3 +334,8 @@ persistence.cache_type=no-cache
 # Installation map URL for installation map iframes. 
 # If blank or commented out, no map will be shown and that page will be hidden.
 # installation_map_url=https://installationmap.netlify.app/
+
+# Turn cache management endpoint on (default) or off
+# cache.endpoint.enabled=false
+# API key for access to cache management endpoint
+# cache.endpoint.api-key=fd15f1ae-66f2-4b8a-8d54-fb899b03557e
```

Comment:
This works as a means to authenticate cache clearing requests, but I don't like that it does not use spring security. I think in a perfect world,  we have a role for clearing the cache, and we use that for security. My guess is that you didn't do that because that pattern fails for portals that do not use authentication. 

I think this all comes down to the fact that portals without authentication still need a way to keep and verify secrets, and as of now, we don't have a standardized way of supporting that. This PR doesn't deserve to be scuttled while we talk about how to standardize this, but we should have that conversation.

Reply:
That crossed my mind, but it will complicate automated import scripts that perform a cache flush at the end. But this might be my ignorance. We can discuss solutions here.